config/config.go

```diff
@@ -365,67 +363,24 @@ func CreateVMConfig(cfg *Config, index int) (*vm.Config, error) {
 	return vmCfg, nil
 }
 
-func checkUnknownFields(data []byte) (string, error) {
-	// While https://github.com/golang/go/issues/15314 is not resolved
-	// we don't have a better way than to enumerate all known fields.
-	var fields = []string{
-		"Name",
-		"Http",
-		"Rpc",
-		"Workdir",
-		"Vmlinux",
-		"Kernel",
-		"Tag",
-		"Cmdline",
-		"Image",
-		"Cpu",
-		"Mem",
-		"Sshkey",
-		"Bin",
-		"Bin_Args",
-		"Debug",
-		"Output",
-		"Hub_Addr",
-		"Hub_Key",
-		"Dashboard_Addr",
-		"Dashboard_Key",
-		"Syzkaller",
-		"Type",
-		"Count",
-		"Devices",
-		"Procs",
-		"Cover",
-		"Reproduce",
-		"Sandbox",
-		"Leak",
-		"Enable_Syscalls",
-		"Disable_Syscalls",
-		"Suppressions",
-		"Ignores",
-		"Initrd",
-		"Machine_Type",
-		"Odroid_Host_Addr",
-		"Odroid_Slave_Addr",
-		"Odroid_Console",
-		"Odroid_Hub_Bus",
-		"Odroid_Hub_Device",
-		"Odroid_Hub_Port",
+func checkUnknownFields(data []byte) error {
+	structType := reflect.ValueOf(Config{}).Type()
+	fields := make(map[string]bool, 0)
```

Comment:
please drop this ", 0"
it's unnecessary and usually not spelled in idiomatic Go